initializr-service/src/main/resources/application.yml

```diff
@@ -1252,6 +1252,33 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+        - name: Codecentric Spring Boot Admin Server
```

Comment:
I think a `versionRange` is in order here (unless there is support for Spring Boot 2 already, in which case I expect a separate version).